pkgs/development/tools/misc/terraform-ls/default.nix

```diff
@@ -1,18 +1,21 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "terraform-ls";
-  version = "0.7.0";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "hashicorp";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1afdd1zs242nh1cync5ip1fbah34wc3gbsx3hwwiisc8yziwiq18";
+    sha256 = "qtnrXQMN/bQb2041W1SkrUjL/G0B2Rc7zcIkDroxDoM=";
   };
+  vendorSha256 = "npoP7wfjDogA7w3m9eenXSZXYFOwfliCWjXcomlJC+0=";
 
   goPackagePath = "github.com/hashicorp/terraform-ls";
 
+  doCheck = false;
```

Comment:
Does upstream have no tests or do they not pass?

Reply:
upstream has tests which fail in sandbox mode because they download stuff from releases.hashicorp.com

Reply:
Could you add this as a comment?